src/lucky/redirectable.cr

```diff
@@ -1,19 +1,68 @@
+# Redirect the request
+#
+# There are multiple ways to redirect inside of an action. The most common ways are to redirect to a `Lucky::Action` class, or a URL/path `String`. Both use the `redirect` method:
+#
+# ```crystal
+# redirect to: Users::Index
+# redirect to: Users::Show.with(user.id)
+# redirect to: "https://luckyframework.org/"
+# redirect to: "/users"
+# ```
+#
+# By default, the method will set the status code to `302` A.K.A. "Found". If you want to customize the status code, you can pass it directly:
+#
+# ```crystal
+# redirect to: Users::Index, status: 301
+#
+# # or use the built in enum value
+# redirect to: Users::Index, status: Status::MovedPermanently
+# ```
+#
+# You can find a list of all of the possible statuses [here](https://github.com/luckyframework/lucky/blob/master/src/lucky/action.cr).
+#
+# Internally, all the different methods in this module eventually use the
+# method that takes a `String`. However, it's recommended you pass a
+# `Lucky::Action` class if possible because it guarantees runtime safety.
 module Lucky::Redirectable
+  # Redirect using a `Lucky::RouteHelper`
+  #
+  # ```crystal
+  # route = Lucky::RouteHelper.new(method: :get, path: "/users/1")
```

Comment:
I think it's pretty rare to instantiate a RouteHelper manually. A RouteHelper is used when you use `MyAction.with(..)`

So this example would be something like `redirect to: Users::Show.with(id)`

Reply:
This is why I'm doing documentation 😄